docs/topic-guides/model-definitions/index.txt

```diff
@@ -9,12 +9,10 @@ application framework (e.g., Keras, TensorFlow), in terms of loading
 training data, describing a model architecture, and specifying the
 underlying iterative optimization training algorithms.
 
-There interface you can use to write model definition is the Trial API.
-With the :ref:`model-definitions_trial-api`, developers expose the
-components of a model definition to Determined’s training loop
-implementation.
-
-The command line interface and Python SDK support the Trial API.
+Model definitions are written using the Trial API, which is an interface
+provided by Determined. With the :ref:`model-definitions_trial-api`,
```

Comment:
As a general rule, let's stick to
```suggestion
provided by Determined. With the :ref:`Trial API <model-definitions_trial-api>`,
```
or something like
```suggestion
provided by Determined. See :ref:`model-definitions_trial-api` to learn more...
```
i.e., either treat the title of the page as a location or provide link text explicitly to be robust against page title changes.

Also, while we're here, the first mention should probably be the link instead, and maybe we don't need another link in the list below (or only link there; either way).

Reply:
Sounds reasonable. I used the first form. I kept both references linked just because it seems like more links is safer than less :)